BuildSimHubAPI/helpers/parametric_job.py

```diff
@@ -9,22 +9,22 @@ class ParametricJob():
     BASE_URL = 'https://my.buildsim.io/'
 
     def __init__(self, userKey, mk, base_url=None):
```

Comment:
`mk` is not consistent between the classes. In `/helpers/parametric_model.py` it is called `model_key`.

Reply:
oh, yes, I will change it and double check other parameters, thanks for pointing it out!